packages/fast-components-react-base/src/image/image.tsx

```diff
@@ -107,7 +107,7 @@ class Image extends Foundation<IImageHandledProps & IManagedClasses<IImageClassN
         const vp: number = breakpoints[`vp${breakpointsId}`];
         const minWidth: number = vp;
 
-        if (!srcset) {
+        if (!srcset || this.props.src) {
```

Comment:
looks like if src exists there is no src element?

Reply:
yea what is the purpose of `this.props.src`? it doesn't seem to drive any attribute, which seems un-intuitive.